com.sap.cloud.lm.sl.cf.client/src/main/java/com/sap/cloud/lm/sl/cf/client/util/ExecutionRetrier.java

```diff
@@ -18,9 +18,15 @@
     private static final long DEFAULT_RETRY_COUNT = 3;
     private static final long DEFAULT_WAIT_TIME_BETWEEN_RETRIES_IN_MILLIS = 5000;
 
+    private boolean failSafe;
     private long retryCount = DEFAULT_RETRY_COUNT;
     private long waitTimeBetweenRetriesInMillis = DEFAULT_WAIT_TIME_BETWEEN_RETRIES_IN_MILLIS;
-
+    
+    public ExecutionRetrier withFailSafe(boolean failSafe) {
```

Comment:
I would've preferred it if the method was called failSafe() without any arguments and just assigned true to the field. It's default value is false anyway.